### ISSUE TITLE
Feature request: module type that runs once on the server

### ISSUE THREAD

**Issue by dma** [subgraph/Vega#54]
It would be nice to have a way to run a module only once (without a hack to do this), like in the pre-beta versions of Vega.

Example modules:

```
To determine if HTTP TRACE is enabled.
To check SSL settings (SSLv2?) <-- this may not even be possible in a Vega module and may need to be a probe written in Java 
```

etc..
